apps/api/views/homework.py

```diff
@@ -32,11 +36,25 @@ class SubmissionViewSet(ModelViewSet):
     serializer_class = SubmissionSerializer
     permission_classes = (SubmissionPermissionCheck,)
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
```

Comment:
why is this overridden? add comment?

Reply:
👍 

Didn't have a reason previously, now I do, added comment to explain.

Reply:
Sorry to be annoying, but if you make this a "docstring" of the function, by putting it in the body like this:

```
def create(self, request, *args, **kwargs):
    """Overwritten so we can return responses or default from post_submission"""
```

then it may show up in people's auto complete and be a useful hint at some point.

Reply:
👍